### PR TITLE
fix: change subgraph url and adjust query to get positions

### LINKED ISSUE #204: Rework EMP tools to work with Perp
Created by David Adams via monday.com integration. 🎉

### PR DESCRIPTION
**Pull request by daywiss**
**motivation**
#204 

**summary**
This enables viewing all positions and for adding deposits and liquidating.  This leverages the work @nicholaspai has done with the uma subgraph and is using his recently updated code which supports the perp on kovan.

**screens**
image

image

**issue**
closes #204

### CODE REVIEW

File: apollo/uma/queries.ts [hunk 1]
```diff
@@ -21,12 +21,8 @@ export const EMP_DATA = gql`
           id
         }
         liquidationId
-        liquidator {
-          address
-        }
-        disputer {
-          address
-        }
+        liquidator
```
Comment (nicholaspai) [inline apollo/uma/queries.ts:24]: I wouldn't merge this in yet until the mainnet subgraph is also fixed. It's currently syncing [here](https://thegraph.com/explorer/subgraph/nicholaspai/mainnet-contracts-staging?version=pending)

File: apollo/client.ts [hunk 2]
```diff
@@ -6,8 +6,9 @@ import {
 } from "@apollo/client";
 import { request } from "http";
 
+// this has the latest subgraph for now, may need to be updated later
 const umaLinkKovan = new HttpLink({
-  uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/uma-kovan",
+  uri: "https://api.thegraph.com/subgraphs/name/nicholaspai/kovan-staging",
 });
 const umaLinkMainnet = new HttpLink({
   uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/uma",
```
Comment (nicholaspai) [inline apollo/client.ts:14]: Once the [mainnet subgraph](https://thegraph.com/explorer/subgraph/nicholaspai/mainnet-contracts-staging?version=pending) is done syncing, you can change this URL to `/nicholaspai/mainnet-contracts-staging`

File: apollo/client.ts [hunk 3]
```diff
@@ -6,8 +6,9 @@ import {
 } from "@apollo/client";
 import { request } from "http";
 
+// this has the latest subgraph for now, may need to be updated later
 const umaLinkKovan = new HttpLink({
-  uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/uma-kovan",
+  uri: "https://api.thegraph.com/subgraphs/name/nicholaspai/kovan-staging",
```
Comment (nicholaspai) [inline apollo/client.ts:11]: I've re-indexed this subgraph at `nicholaspai/kovan-contracts-staging` and will deprecate `kovan-staging` going forwards

Comment (mrice32) [reply apollo/client.ts:11]: Just out of curiosity, why isn't this under an UMA namespace?

**Review by mrice32 — APPROVED** [review]
LGTM! Just curious why we're not pointing at uma namespaced subgraphs?

File: apollo/client.ts (hunk 3 above)
Comment (nicholaspai) [inline apollo/client.ts:11]:
```suggestion
  uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/kovan-contracts",
```

File: apollo/client.ts (hunk 2 above)
Comment (nicholaspai) [inline apollo/client.ts:14]:
```suggestion
  uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/mainnet-contracts",
```

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**